alembic_offline/api.py

```diff
@@ -158,3 +158,35 @@ def get_script_data(script_directory, file_name):
     with codecs.open(script_file_name, encoding='utf-8') as fd:
         script_text = fd.read()
     return dict(type=script_type, script=script_text, path=file_name)
+
+
+def generate_migration_graph(config, label_callback=None):
```

Comment:
there's no way to generate the diagram without calling this function from python
can you add a commandline interface for it? it can be `alembic-offline graph` subcommand